### DISCUSSION TITLE
Securing Web GUI using Let's Encrypt

### DISCUSSION THREAD

**Discussion by jackc94** [koush/scrypted#331] [Q&A] [answered]
Hi all, 

I'd really like to secure my Scrypted Web UI using a Let's Encrypt SSL cert but need some assistance... could anybody provide a step by step for this on a Raspberry Pi local install? Happy to change to a Docker install if that makes it easier but nothing else runs on that Pi other than Scrypted so thought I'd cut out the middle man. 

Thanks in advance for any assistance.

**Comment by koush** [comment, marked as answer]
ssl cert usage is on the wiki. but use a reverse proxy like nginx, etc, for lets encrypt imo.

**Reply by jackc94** [reply]
That’s what I ended up doing in the end… the only thing I don’t like about that is that the original, unsecured UI is still accessible.
